queries.sql

```diff
@@ -1,3 +1,5 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
```

Comment:
![image](https://github.com/fpsapc/vetClinic/assets/93039359/de422a76-33d9-4b1c-9d0b-9cb1ee7a94f2)

- [ ] It would be nice if you follow the project requirements and add the missing queries in the queries.sql file 👍